### ISSUE TITLE
Query parameters broken for string parameters

### ISSUE THREAD

**Issue by rushton** [trinodb/trino-python-client#48]
The following assertion fails:
```python
      cur.execute(
          """
          select * from (
              values
              (1, 'one', 'a'),
              (2, 'two', 'b'),
              (3, 'three', 'c'),
              (4, 'four', 'd'),
              (5, 'five', 'e')
          ) x (id, name, letter)
          where name >= ?
          """,
          params=("one",),  # expecting all the rows with id >= 3
      )

      rows = cur.fetchall()

      assert len(rows) == 1
      assert rows[0][1] == "one"
```
with the following error:

```
presto.exceptions.PrestoUserError: PrestoUserError(type=USER_ERROR, name=EXPRESSION_NOT_CONSTANT, message="line 1:51: Constant expression cannot contain column references", query_id=20201014_222020_00005_ssyxh)
```

the generated statement is incorrect because it does not quote the string argument properly:

```sql
EXECUTE st_0c49e903f9cb4331ab305ac778bbae43 USING one
```

should be:
```sql
EXECUTE st_0c49e903f9cb4331ab305ac778bbae43 USING 'one'
```

**Comment by rushton** [comment]
possibly related to #45, a result of changes made in #41

**Comment by ebyhr** [comment]
Already fixed in #50
